### PR TITLE
Makes secondary enrollable without completing CSA.

### PR DESCRIPTION
**Pull request by Caleb-T-Owens**
## Status

* Current Status: Ready for (front-end / tech) review
* Review App link(s): *populate this with links to the relevant parts of the review app*

## Review progress:

- [x] Browser tested
- [x] Front-end review completed
- [x] Tech review completed

## What's changed?

This PR also removes the old legacy activities which users can no longer complete, and supporting logic around them

## Steps to perform after deploying to production

- remove 
`bin/rails schedule_cs_accelerator_eligible_courses_for_secondary_certificate_users` from heroku schedulers

### CODE REVIEW

**Review by dcorking — APPROVED** [review]
Nice cleanup alongside the functionality change . LGTM 🚂

File: spec/components/pathway_enrol_component_spec.rb [hunk 1]
```diff
@@ -39,9 +39,11 @@
     end
 
     context 'when user doesn\'t meet enrolment requirements' do
-      let(:programme) { create(:secondary_certificate) }
-
       it 'doesn\'t render an Enrol button' do
+        allow(programme).to receive(:user_is_eligible?).with(user).and_return(false)
```
Comment (dcorking) [inline spec/components/pathway_enrol_component_spec.rb:43]: nice fix 😄

**Review by dcorking — APPROVED** [review]
LGTM 🚂

File: spec/components/pathway_enrol_component_spec.rb (hunk 1 above)
Comment (Caleb-T-Owens) [reply spec/components/pathway_enrol_component_spec.rb:43]: Thanks!

**Review by dcorking — APPROVED** [review]
LGTM 🚂

**Pull request merged**